src/modules/dashboard/components/Extensions/ExtensionStatus.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import { ColonyExtension } from '~data/index';
+import Tag from '~core/Tag';
+
+const MSG = defineMessages({
+  installed: {
+    id: 'dashboard.Extensions.ExtensionStatus.installed',
+    defaultMessage: 'Installed',
+  },
+  notInstalled: {
+    id: 'dashboard.Extensions.ExtensionStatus.notInstalled',
+    defaultMessage: 'Not installed',
+  },
+  missingPermissions: {
+    id: 'dashboard.Extensions.ExtensionStatus.missingPermissions',
+    defaultMessage: 'Missing permissions',
+  },
+  deprecated: {
+    id: 'dashboard.Extensions.ExtensionStatus.deprecated',
+    defaultMessage: 'Deprecated',
+  },
+  enabled: {
+    id: 'dashboard.Extensions.ExtensionStatus.enabled',
+    defaultMessage: 'Enabled',
+  },
+  notEnabled: {
+    id: 'dashboard.Extensions.ExtensionStatus.notEnabled',
+    defaultMessage: 'Not enabled',
+  },
+});
+
+interface Props {
+  installedExtension?: ColonyExtension | null;
+}
+
+const ExtensionStatus = ({ installedExtension }: Props) => {
+  let status;
+  let theme;
+
+  if (!installedExtension) {
+    status = MSG.notInstalled;
+  } else if (!installedExtension.details.initialized) {
+    status = MSG.notEnabled;
+  } else if (installedExtension.details.missingPermissions.length) {
+    status = MSG.missingPermissions;
+    theme = 'danger';
+  } else if (installedExtension.details.deprecated) {
+    status = MSG.deprecated;
+    theme = 'danger';
+  } else if (installedExtension.details.initialized) {
+    status = MSG.enabled;
+    theme = 'primary';
+  } else {
+    status = MSG.installed;
+  }
+  return <Tag appearance={{ theme }} text={status} />;
```

Comment:
Was the tooltip copy over the extension status tag deprecated?

![Screenshot from 2020-11-17 14-33-12](https://user-images.githubusercontent.com/1193222/99391001-2b5a7300-28e2-11eb-9d20-928fe33e6c6a.png)